src/resources/url/url.controllers.js

```diff
@@ -39,11 +40,12 @@ const createUrl = async (req, res) => {
     : null
 
   // Save
+  const encodedPasscode = encodePasscode(passcode)
   const doc = new Url({
     _id,
     url,
     expires,
-    passcode
+    passcode: encodedPasscode,
```

Comment:
nit: since we don't use `encodedPasscode` anywhere else, might be cleaner to write this inline as `passcode: encodePasscode(passcode)`